### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by Rxann** [pmndrs/react-spring#1932] [Support]
Hi,
I am using react-spring parallax and I was wondering if I can use images in a ParallaxLayer? I am using preact and I tried to add a backround Image and that didn't work so I was wondering if I could just add an <img> tag to a layer.

**Comment by kindoflew** [comment]
Hi!

You should be able to add a background image to a `ParallaxLayer`. Could you share a reproduction (maybe in codesandbox or something similar?)

You can also nest an `img` tag inside a `ParallaxLayer`:

```jsx
<ParallaxLayer>
  <img src='some-img.png' alt='my image' />
</ParallaxLayer>
```

**Reply by Rxann** [reply]
Hi, I've tried this and it works perfectly!
Thank you so much!
